Platform/RaptorlakeBoardPkg/CfgData/CfgDataInt_Rpls_Rvp_Ddr5.dlt

```diff
@@ -14,9 +14,10 @@
 #
 
 PLATFORMID_CFG_DATA.PlatformId           | 0x0019
-PLAT_NAME_CFG_DATA.PlatformName          | 'RPL_S'
+PLAT_NAME_CFG_DATA.PlatformName          | 'RPLS S14'
 GEN_CFG_DATA.PayloadId                   | ''
-FEATURES_CFG_DATA.Features.S0ix          | 0x1
+FEATURES_CFG_DATA.Features.S0ix          | 0x0
```

Comment:
Is it cause by missing the CfgData_Tcc_Feature.dlt in cfgdata folder?
Turn off the S0ix by default is expected on non-ecc sku?
On release build, is BootToShell a default feature?

Reply:
I think this change is just aligned with ADL platform.